pkg/portallocator/portallocator_test.go

```diff
@@ -626,10 +637,18 @@ func countTotalAllocatedPorts(pa *PortAllocator) int {
 	return count
 }
 
-func countTotalPorts(pa *PortAllocator) int {
+func countTotalPorts(pa *portAllocator) int {
 	count := 0
 	for _, node := range pa.portAllocations {
 		count += len(node)
 	}
 	return count
 }
+
+func portAllocatorImpl(t *testing.T, pai Interface) *portAllocator {
+	pa, ok := pai.(*portAllocator)
+	if !ok {
+		t.Fatalf("toPortAllocator() called on portAllocator of type %t", pai)
```

Comment:
Non blocking nit, only change if you want: We do tend to use testify,so `require.Failf(...)` would be inline with that (not sure if I missed these in the previous PR).

Reply:
I changed a couple from my previous PR and will scan for others here before I re-upload.